pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
```

Comment:
Previously `bokeh` was relying on the legacy behaviour, but given the proposed changes, it should be safe to fully adhere to PEP 517.